citations-tool/tool/src/webapp/js/citationscript.js

```diff
@@ -671,11 +671,34 @@ function checkRequiredFields( alertMsg ) {
     return false;
   }
   
+  //If electronic citation check if at least one url is supplied
+  if($('#type_selector').val() == 'electronic'){
+      var showUrlAlert = $("input:text[name^='url_']" ).filter(function(){
+                         return this.value.trim().length
+                         }).length > 0;
+      if(!showUrlAlert){
```

Comment:
I just thought the situation where user has multiple urls for the link field and while editing citation if they remove the first url and keep the second one, I think I overcomplicated it.